### ISSUE TITLE
SS analysis

### ISSUE THREAD

**Issue by ryanmcgreevy** [ryanmcgreevy/ModelMaker#50]
Reminder for myself: SS analysis issue, gnuplot errors:

`
QXcbConnection: Could not connect to display 
Could not connect to existing gnuplot_qt. Starting a new one.
QXcbConnection: Could not connect to display 

Warning: slow font initialization
`
Causes analysis to terminate early. Tested on linux. Test if this happens on OSX.

**Comment by ryanmcgreevy** [comment]
I'm adding @jribeiroUIUC to this issue because we spoke about this briefly in the developer meeting. This is the error I'm getting from gnuplot on my linux machine. On OSX I'm getting a segfault and haven't investigated further. I haven't yet dug into the ss_analysis code further to see what exact step is causing this issue.

**Comment by jribeiroUIUC** [comment]
I did try to replicate the error and I was not able. I am using the gnuplot version "5.0 patchlevel rc2".
John suggested that I should move the "reset" and "clear" commands to the end of the script. I did that, but I cannot test if that worked since I cannot replicate the errors. Could you see if that is still returning any issue to you?

**Comment by ryanmcgreevy** [comment]
Yep that seemed to work! Thanks! I have not tested it yet on OSX to see if it was the same issue there, but it now works on linux at least.

**Comment by jribeiroUIUC** [comment]
I actually deleted the commands reset and clear which were creating a new unnecessary plot. But I believe that I messed up in the process of pushing my changes. Can you revert the changes that I made to the modelmake.tcl? By the way, can you please check if the new ss_analysis.tcl also works for you? Thanks and sorry about that

**Comment by ryanmcgreevy** [comment]
Hmm, without those commands, the .ps file that is created won't open properly on my linux machine using imagemagick or evince, so I'm putting them back in for now.
